microsoft-edge/webview2/gettingstarted/winUI.md

```diff
@@ -0,0 +1,293 @@
+  
+    * **Project name (Package)**: This is a [Windows Application Packaging Project](https://docs.microsoft.com/en-us/windows/msix/desktop/desktop-to-uwp-packaging-dot-net) that is configured to build the app into an MSIX package for deployment. This project contains the [package manifest](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/schema-root) for your app, and it is the startup project for your solution by default.
+
+    
+    
+    
+8. Double click the MainWindow.xml file in the Solution Explorer to view the code. Click F5 to build the project. The project should run a small example of a button in a window.
+    
+## Step 2 - Create WebView
+
+Next lets add a WebView to your application.  
+
+1.  Open `MainWindow.xaml`.  Add the WebView2 XAML namespace by inserting the following line inside the `<Window/>` tag.  
+    
+    ```xml
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+    ```  
+    
+    Confirm that the code in `MainWindow.xaml` looks like the following code snippet.  
+    
+    ```xml
+    <Window
+          x:Class="WinUI_Sample.MainWindow"
+          xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+          xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+          xmlns:local="using:WinUI_Sample"
+          xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+          xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+          mc:Ignorable="d"
+          xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+          >
+        
+          <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
+            <Button x:Name="myButton" Click="myButton_Click">Click Me</Button>
+          </StackPanel>
+
+    </Window>
+    ```  
+    
+1.  Add the WebView2 control by replacing the `<StackPanel>` tags, with the following code snippet.  The `Source` property sets the initial URI displayed in the WebView2 control.  
+    
+    ```xml  
+    <Grid>
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+
+        <controls:WebView2 x:Name="MyWebView"  controls:Grid.Row="1" controls:Grid.ColumnSpan="2" 
+        Source="https://www.microsoft.com" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" />
+
+    </Grid>
+    ```  
+3.  Additionally, open `MainWindow.xaml.cs` and comment out the following line:
```

Comment:
```suggestion
3.  Additionally, open `MainWindow.xaml.cs` and comment out the following line for now:
```

I like this as a hint to not just delete the function, since it will be used again later.